assets/sass/elements/_forms.scss

```diff
@@ -20,6 +20,11 @@ fieldset {
   width: 100%;
 }
 
+// Hack to let legends or elements within legends have margins in webkit browsers
```

Comment:
This is quite hacky but is the least hacky solution I could find for fixing an issue in all webkits which will not allow a margin on the form hint text. I suspect this might be too controversial (a similar solution was tried in #76 by @timpaul and rejected). That's why I created an alternative solution to fix the same problem with #485.